static/admin/config.yml

```diff
@@ -127,6 +127,7 @@ collections:
             - { label: "🇱🇹 Lietuviškai (official)", value: "lt" }
             - { label: "🇪🇸 Spanish", value: "es" }
             - { label: "🇹🇷 Turkish", value: "tr" }
+            - { label: "Bn Bangla", value: "bn" }
```

Comment:
I think i made any mistake here . Please correct me if i wrong here .

Reply:
The value looks okay, it’s just the flag i am not sure about